### PR TITLE
Support GHC 8.10

### PR DESCRIPTION
**Pull request by jberryman**

### CODE REVIEW

File: src/GHC/HeapView.hs [hunk 1]
```diff
@@ -228,6 +228,9 @@ ppClosure showBox prec c = case c of
     MutArrClosure {..} -> app
         --["toMutArray", "("++show (length mccPayload) ++ " ptrs)",  intercalate "," (shorten (map (showBox 10) mccPayload))]
         ["[", intercalate ", " (shorten (map (showBox 10) mccPayload)),"]"]
+    SmallMutArrClosure {..} -> app
+        --["toMutArray", "("++show (length mccPayload) ++ " ptrs)",  intercalate "," (shorten (map (showBox 10) mccPayload))]
+        ["[", intercalate ", " (shorten (map (showBox 10) mccPayload)),"]"]
```
Comment (nomeata) [inline src/GHC/HeapView.hs:233]: This would break ealier GHC versions, right?

**Comment by jberryman** [comment]
@nomeata seems github / travis are being flaky and CI has stopped running. Hopefully latest is finally working (but I woudn't be surprised if not).

**Comment by jberryman** [comment]
@nomeata okay, done fiddling!

**Comment by nomeata** [comment]
Thanks!

**Pull request merged**